libs/domain/product/src/services/category/adapter/default-product-category.adapter.ts

```diff
@@ -12,7 +16,7 @@ export class DefaultProductCategoryAdapter implements ProductCategoryAdapter {
     protected transformer = inject(JsonAPITransformerService)
   ) {}
 
-  get(categoryId: string): Observable<ProductCategory> {
+  get(qualifier: ProductCategoryQualifier): Observable<ProductCategory> {
```

Comment:
this should not be a string in a first place agreed, but is it a breaking change?

ah, I guess we are missing string handling for <1.4?